jenkins/PerfCI_OpenShift_Deployment/Jenkinsfile

```diff
@@ -28,11 +28,25 @@ pipeline {
         stage('Cleanup') {
             steps {
                 script {
-                    sh '''
+                    try {
+                        // Delete benchmark-runner images
+                        sh '''
                         if [[ "$(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null)" != "" ]]; then
                             sudo podman rmi -f $(sudo podman images -q quay.io/ebattat/benchmark-runner 2> /dev/null)
                         fi
-                    '''
+                        '''
+                        // Stop running containers
+                        sh 'sudo podman ps | awk \'{print $1}\' | xargs -I % sudo podman stop % 2>/dev/null'
```

Comment:
It would be simpler to do

```
'sudo podman ps | awk \'{print $1}\' | sudo xargs -I podman stop 2>/dev/null'
```
Please test this; it definitely makes it more readable if it works.

Reply:
I will try it, it insides Jenkinsfile so not all working the same

Reply:
Fixed